bigbluebutton-html5/imports/ui/components/settings/container.jsx

```diff
@@ -33,72 +33,71 @@ class SettingsMenuContainer extends Component {
-    this.setState({ currentFontSize: fs }, function () {
-      this.handleApplyFontSize();
-    });
+  handleGetFontSizeName() {
+    return this.fontControl.properties[this.state.currentFontSize].name;
+  };
+
+  handleApplyFontSize() {
+    const size = this.fontControl.properties[this.state.currentFontSize].size;
+    document.getElementsByTagName('html')[0].style.fontSize = size;
+  };
+
+  handleIncreaseFontSize() {
+    let fs = ( this.state.currentFontSize < MAX_FONTSIZE) ? ++this.state.currentFontSize : MAX_FONTSIZE;
+    LocalStorage.setItem('bbbFontSize', fs);
+    this.setState({ currentFontSize: fs }, function () {
+      this.handleApplyFontSize();
+    });
+  };
+
+  handleDecreaseFontSize() {
+    let fs = ( this.state.currentFontSize > MIN_FONTSIZE) ? --this.state.currentFontSize : MIN_FONTSIZE;
+    LocalStorage.setItem('bbbFontSize', fs);
+    this.setState({ currentFontSize: fs }, function () {
+      this.handleApplyFontSize();
+    });
+  };
+
+  handleSaveFontState() {
+    let fs = LocalStorage.getItem('bbbFontSize') || DEFAULT_FONTSIZE;
+    LocalStorage.setItem('bbbSavedFontSize', fs);
+    LocalStorage.setItem('bbbSavedFontSizePixels', this.fontControl.properties[fs].size);
+    this.setState({ currentFontSize: fs }, function () {
+      this.handleApplyFontSize();
+    });
+  };
+
+  handleRevertFontState(){
+    let fs = LocalStorage.getItem('bbbSavedFontSize') || DEFAULT_FONTSIZE;
+    this.setState({ currentFontSize: fs }, function () {
+      this.handleApplyFontSize();
+    });
   };
 
-
   render() {
 
-    const handleGetFontSizeName = () => this.handleGetFontSizeName();
-    const handleIncreaseFontSize = () => this.handleIncreaseFontSize();
-    const handleDecreaseFontSize = () => this.handleDecreaseFontSize();
-    const handleSaveFontState = () => this.handleSaveFontState();
+    const handleGetFontSizeName = () => this.handleGetFontSizeName();
+    const handleIncreaseFontSize = () => this.handleIncreaseFontSize();
+    const handleDecreaseFontSize = () => this.handleDecreaseFontSize();
+    const handleSaveFontState = () => this.handleSaveFontState();
     const handleRevertFontState = () => this.handleRevertFontState();
-
-    return (
+
+    return (
       <Settings
         handleGetFontSizeName={handleGetFontSizeName}
         handleIncreaseFontSize={handleIncreaseFontSize}
         handleDecreaseFontSize={handleDecreaseFontSize}
         handleSaveFontState={handleSaveFontState}
         handleRevertFontState={handleRevertFontState}
-        {...this.props}>
-        {this.props.children}
-      </Settings>
-    );
-  }
-}
-
-export default createContainer(() => {
-  let data = Service.checkUserRoles();
-  return data;
-}, SettingsMenuContainer);
+        {...this.props}>
+        {this.props.children}
+      </Settings>
+    );
+  }
+}
+
+export default createContainer(() => {
+  let data = Service.checkUserRoles();
+  return data;
```

Comment:
Hey @Gcampes , I didn't touch this component, just switched the line endings from Windows to Unix.
Maybe it would work :-)